ingest-router/src/locale.rs

```diff
@@ -98,8 +99,8 @@ impl Locales {
     }
 
     /// Get the cells for a specific locale
-    pub fn get_cells(&self, locale: &str) -> Option<&Cells> {
-        self.locale_to_cells.get(locale)
+    pub fn get_cells(&self, locale: &str) -> Option<Arc<Cells>> {
```

Comment:
Switched to `Arc<Cells>` since we pass `Cells` to a bunch of places now - and it's easier for the data to be owned